### PR TITLE
feat: add teaser ets couplings

### PR DESCRIPTION
**Pull request by macintoshpie**
#### Any background context you want to provide?
Teaser only had couplings to ETS Stubs

#### What does this PR accomplish?
Adds Teaser <> ETS heating and cooling indirect couplings

#### How should this be manually tested?
Run the following tests, open the generated packages in dymola and verify everything looks correct

```bash
poetry run pytest ./tests/model_connectors/test_teaser_heating.py
poetry run pytest ./tests/model_connectors/test_teaser_cooling.py
poetry run pytest ./tests/model_connectors/test_teaser_district_heating_and_cooling_systems.py
```

#### What are the relevant tickets?
#380 

#### Screenshots (if appropriate)
Screen Shot 2021-05-05 at 2 12 50 PM
Screen Shot 2021-05-05 at 1 47 48 PM
Screen Shot 2021-05-05 at 1 45 54 PM

### CODE REVIEW

File: geojson_modelica_translator/model_connectors/load_connectors/teaser.py [hunk 1]
```diff
@@ -629,10 +629,10 @@ def post_process(self, scaffold, keep_original_models=False):
...
         # now create the Package level package. This really needs to happen at the GeoJSON to modelica stage, but
         # do it here for now to aid in testing.
-        pp = PackageParser.new_from_template(
-            scaffold.project_path, scaffold.project_name, ["Loads"]
-        )
-        pp.save()
+        package = PackageParser(scaffold.project_path)
+        if 'Loads' not in package.order:
+            package.add_model('Loads')
+            package.save()
```
Comment (macintoshpie) [inline geojson_modelica_translator/model_connectors/load_connectors/teaser.py:635]: this change avoids overwriting existing package

**Review by nllong — APPROVED** [review]
Looks great. There was a missing parens which I added in [this commit](https://github.com/urbanopt/geojson-modelica-translator/pull/381/commits/4eddf4963a8d86f416754ee3f567940c833122c6). This wasn't causing an issue for running the model, but Dymola was throwing some syntax errors.

This looks amazing btw--super baller! Good work!

**Comment by macintoshpie** [comment]
@nllong thanks! Do we want @kbenne to take a look as well before merging since he helped with the spawn couplings?

**Comment by nllong** [comment]
I say we merge this. It isn't spawn based which is @kbenne 's stomping ground.

**Pull request merged**